api/cloudformation.yml

```diff
@@ -13,17 +13,26 @@ Parameters:
     Type: String
     Default: RS256
 
+  DomainName:
+    Type: String
+
+  CertificateArn:
+    Type: String
+
 Outputs:
 
   Url:
-    Value: !Sub "https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/${Stage}/"
+    Value: !Sub "https://${DomainName}/ui"
```

Comment:
Do we want to return a reference to the UI endpoint, or root?

Reply:
UI was the one I wanted all the time, so I thought I'd save myself the work.

Now that we'll have static domain names for the two deployments this output parameter will probably only be leveraged by ad-hoc deployments, in which case I'll want to straight to the UI for testing I think.

Reply:
although ad-hoc deployments probably won't leverage, `https://${DomainName}`, they'll go straight to the gateway at `https://${RestApi}.execute-api.${AWS::Region}.amazonaws.com/${Stage}/`